[supervisord]
nodaemon=true

[program:sd_func_invoker]
command=python manage.py sd_func_invoker
autostart=true
autorestart=true
redirect_stderr=true
stdout_logfile=/var/log/su.out.log
stderr_logfile=/var/log/su.err.log

[program:sd_func_invoker_delay]
command=python manage.py sd_func_invoker_delay
autostart=true
autorestart=true
redirect_stderr=true
stdout_logfile=/var/log/su.out.log
stderr_logfile=/var/log/su.err.log

[program:image_func_invoker_delay]
command=python manage.py image_func_invoker_delay
autostart=true
autorestart=true
redirect_stderr=true
stdout_logfile=/var/log/su.out.log
stderr_logfile=/var/log/su.err.log

[program:tts_func_invoker_delay]
command=python manage.py tts_func_invoker_delay
autostart=true
autorestart=true
redirect_stderr=true
stdout_logfile=/var/log/su.out.log
stderr_logfile=/var/log/su.err.log

[program:image_func_invoker1]
command=python manage.py image_func_invoker
autostart=true
autorestart=true
redirect_stderr=true
stdout_logfile=/var/log/su.out.log
stderr_logfile=/var/log/su.err.log

[program:image_func_invoker2]
command=python manage.py image_func_invoker
autostart=true
autorestart=true
redirect_stderr=true
stdout_logfile=/var/log/su.out.log
stderr_logfile=/var/log/su.err.log

[program:image_func_invoker3]
command=python manage.py image_func_invoker
autostart=true
autorestart=true
redirect_stderr=true
stdout_logfile=/var/log/su.out.log
stderr_logfile=/var/log/su.err.log

[program:image_func_invoker4]
command=python manage.py image_func_invoker
autostart=true
autorestart=true
redirect_stderr=true
stdout_logfile=/var/log/su.out.log
stderr_logfile=/var/log/su.err.log

[program:image_func_invoker5]
command=python manage.py image_func_invoker
autostart=true
autorestart=true
redirect_stderr=true
stdout_logfile=/var/log/su.out.log
stderr_logfile=/var/log/su.err.log

[program:image_func_invoker6]
command=python manage.py image_func_invoker
autostart=true
autorestart=true
redirect_stderr=true
stdout_logfile=/var/log/su.out.log
stderr_logfile=/var/log/su.err.log

[program:tts_func_invoker1]
command=python manage.py tts_func_invoker
autostart=true
autorestart=true
redirect_stderr=true
stdout_logfile=/var/log/su.out.log
stderr_logfile=/var/log/su.err.log

[program:tts_func_invoker2]
command=python manage.py tts_func_invoker
autostart=true
autorestart=true
redirect_stderr=true
stdout_logfile=/var/log/su.out.log
stderr_logfile=/var/log/su.err.log

[program:tts_func_invoker3]
command=python manage.py tts_func_invoker
autostart=true
autorestart=true
redirect_stderr=true
stdout_logfile=/var/log/su.out.log
stderr_logfile=/var/log/su.err.log

[program:tts_func_invoker4]
command=python manage.py tts_func_invoker
autostart=true
autorestart=true
redirect_stderr=true
stdout_logfile=/var/log/su.out.log
stderr_logfile=/var/log/su.err.log

[program:tts_func_invoker5]
command=python manage.py tts_func_invoker
autostart=true
autorestart=true
redirect_stderr=true
stdout_logfile=/var/log/su.out.log
stderr_logfile=/var/log/su.err.log

[program:tts_func_invoker5]
command=python manage.py tts_func_invoker
autostart=true
autorestart=true
redirect_stderr=true
stdout_logfile=/var/log/su.out.log
stderr_logfile=/var/log/su.err.log